ajax/libs/arraysweeper/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "arraysweeper",
+  "filename": "arraysweeper.min.js",
+  "version": "0.1.0",
```

Comment:
Please remove the `version` field. It's not needed when you're adding a new library via a single package.json file.